content/en/cloudcraft/advanced/find-id-using-api.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Find Cloud Account or Team ID using our API
+kind: guide
+---
+
+## Overview
+
+Currently, the Cloudcraft UI doesn't expose the ID of your AWS or Azure accounts or teams. However, you can still find these IDs using our API and a bit of manual work.
+
+## Prerequisites
+
+Before you begin, make sure you have the following:
+
+- A Cloudcraft user with the [Owner or Administrator role][1].
+- A [Cloudcraft Pro subscription][2].
+- A Unix-like environment, such as Linux, macOS, or WSL on Windows with cURL and [jq][3] installed.
+- A basic understanding of the command-line interface.
+- A basic understanding of how to use APIs.
+
+You must also have at least one AWS or Azure account added to Cloudcraft.
+
+## Finding the cloud account ID
+
+Finding the ID of your AWS or Azure account is easy; you can do it by making a single API call.
+
+To find the ID of your cloud account, run the following command:
+
+{{< code-block lang="shell" >}}
+curl \
+  --url 'https://api.cloudcraft.co/${CLOUD}/account' \
+  --tlsv1.2 \
+  --proto '=https' \
+  --compressed \
+  --silent \
+  --header "Authorization: Bearer ${API_KEY}" | jq .
+{{< /code-block >}}
+
+Replace `CLOUD` with `aws` or `azure` and `API_KEY` with your Cloudcraft API key.
+
+The response should look something like this for AWS:
+
+{{< code-block lang="json" filename="cloudcraft-aws-response.json" >}}
+{
+  "accounts": [
+    {
+      "id": "8bfc6773-7fa2-49b3-8016-5e0e9a2e2aff",
```

Comment:
```suggestion
      "id": "8bfc6773-7fa2-49b3-8016-5e0e9a2e2aff",
```
If these IDs are sensitive at all (I'm assuming they're not), can also just use `<CLOUD_ACCOUNT_ID>` instead of the actual ID here. The same goes for `externalId`.

Reply:
Those are randomly generated.